CMakeLists.txt

```diff
@@ -4,8 +4,6 @@ PROJECT (ESCARGOT)
 
 # CONFIGURATION
 SET (CMAKE_VERBOSE_MAKEFILE true)
-# so(dll)s are should placed on CMAKE_BINARY_DIR
-SET (CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
```

Comment:
I'm wondering why output libraries should be located in the binary directory here.